include/glow/Backend/BackendUtils.h

```diff
@@ -120,5 +120,23 @@ class RuntimeBundle {
 };
 } // namespace runtime
 
+/// If \p PH is an output placeholder, \returns true.
+/// This is determined by checking if the PH has a user which uses the PH as an
+/// overwritten input.
+bool isOutput(const Placeholder *PH);
+
+/// If \p PH is an input placeholder, \returns true.
+bool isInput(const Placeholder *PH);
+
+/// Convert placeholders to be ordered as input|inputOutput|output|neither.
+/// Packed into {Placeholder *, isInput, isOutput}.
+std::vector<std::tuple<const Placeholder *, bool, bool>>
+getContiguousPlaceHolder(std::vector<const Placeholder *> const &holders);
```

Comment:
Try to have only one implementation of these two functions. Either call one from the other or may be make this function a template-function?

Reply:
> Try to have only one implementation of these two functions. Either call one from the other or may be make this function a template-function?

OK. I merged them into a template-function